www-apps/otrs/Manifest

```diff
@@ -2,3 +2,4 @@ DIST otrs-5.0.25.tar.bz2 20671315 BLAKE2B 045037411039f480434e7c36ab84b63ec0c119
 DIST otrs-6.0.3.tar.bz2 24314514 BLAKE2B 97d794d59269d354b28003879bf0d30cc9b0f7a7d375416df90d85b694ac59e84978fe61183aa058497da18631513c25ecc93e3150f662e2e2efd335014e1e75 SHA512 d7f551e2c9bbe8b303d3de1f7993d76ea22cfd876b61941cfb72a25f2ba7e0a889c275c87aa40c104c1f2e6e1923b7a530cd088bd11b7b575a608b623e2b0541
```

Comment:
The `Bug` tag belongs in footer, not on the first line. Also, please indicate to which version you are bumping it on the summary line.

Reply:
Which footer do you refer to?

Reply:
Last block of the commit message.